### PR TITLE
KYC e2e business flow tests

### PR DESCRIPTION
**Pull request by lizonp**

### CODE REVIEW

File: app/components/kyc/__tests__/KYCFlowCorporationManual.spec.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  zipCode: "00000",
+  isPoliticallyExposed: "true",
+  isUsCitizen: "false",
+  hasHighIncome: "false",
+};
+
+const password = "such-strong-password";
+
+const goToCorporationFlow = () => {
+  cy.visit("/register");
```
Comment (krzkaczor) [inline app/components/kyc/__tests__/KYCFlowCorporationManual.spec.e2e.ts]: we have already covered account creation in `registerWithLightWallet` function. Please just reuse it

Comment (krzkaczor) [reply app/components/kyc/__tests__/KYCFlowCorporationManual.spec.e2e.ts]: also please fix in all other places in your new tests that the same problems occurs.

File: app/components/walletSelector/light/__tests__/LightWalletRegister.spec.e2e.ts [hunk 2]
```diff
@@ -7,12 +11,12 @@ export const registerWithLightWallet = (email: string, password: string) => {
   cy.get(tid("wallet-selector-register-password")).type(password);
   cy.get(tid("wallet-selector-register-confirm-password")).type(password);
   cy.get(tid("wallet-selector-register-button")).click();
-
-  cy.url().should("contain", "/dashboard");
+  cy.wait(5000).visit(appRoutes.dashboard);
```
Comment (krzkaczor) [inline app/components/walletSelector/light/__tests__/LightWalletRegister.spec.e2e.ts]: why would you need it? Redirection is automatic.

File: app/components/kyc/__tests__/KYCFlowCorporationManual.spec.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+const uploadSupportingDocuments = () => {
+  const dropEvent = {
+    dataTransfer: {
+      files: [],
+    },
+  };
+
+  cy.fixture("example.png").then(picture => {
+    return Cypress.Blob.base64StringToBlob(picture, "image/png").then((blob: never) => {
```
Comment (krzkaczor) [inline app/components/kyc/__tests__/KYCFlowCorporationManual.spec.e2e.ts]: why would you type blob as `never`?

File: app/components/kyc/__tests__/KYCFlowPartnershipManual.spec.e2e.ts [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+const uploadSupportingDocuments = () => {
+  const dropEvent = {
+    dataTransfer: {
+      files: [],
+    },
+  };
+
+  cy.fixture("example.png").then(picture => {
+    return Cypress.Blob.base64StringToBlob(picture, "image/png").then((blob: never) => {
```
Comment (krzkaczor) [inline app/components/kyc/__tests__/KYCFlowPartnershipManual.spec.e2e.ts]: same

File: app/components/walletSelector/light/__tests__/LightWalletRegister.spec.e2e.ts (hunk 2 above)
Comment (lizonp) [reply app/components/walletSelector/light/__tests__/LightWalletRegister.spec.e2e.ts]: Sometimes I had issues with login screen. After logging in it stucked on "Confirm your wallet ownership" screen.

File: app/components/kyc/__tests__/KYCFlowCorporationManual.spec.e2e.ts (hunk 3 above)
Comment (lizonp) [reply app/components/kyc/__tests__/KYCFlowCorporationManual.spec.e2e.ts]: Because it's is the only accepted type that didn't throw error;

Comment (krzkaczor) [reply app/components/kyc/__tests__/KYCFlowCorporationManual.spec.e2e.ts]: It's connected with `dropEvent` inferred type. dropEvent.dataTransfer.files  is inferred as `never[]`. Simple fix is to force cast it to any.

**Pull request merged**